### PR TITLE
Change TRACING_SET to static

### PR DESCRIPTION
**Pull request by insipx**
if TRACING_SET is const, `set_global_default` does not work , as each invocation of `sp_io::init_tracing` will try to re-initialize the global logger, creating panics in wasm compiled with `with-tracing` flags. `const` copies while `static` is a unique location.

### CODE REVIEW

**Review by mattrutherford — APPROVED** [review]

File: primitives/io/src/lib.rs [hunk 1]
```diff
@@ -1093,7 +1093,7 @@ mod tracing_setup {
 	};
 	use super::{wasm_tracing, Crossing};
 
-	const TRACING_SET : AtomicBool = AtomicBool::new(false);
+	static TRACING_SET : AtomicBool = AtomicBool::new(false);
```
Comment (bkchr) [inline primitives/io/src/lib.rs:1096]:
```suggestion
	static TRACING_SET: AtomicBool = AtomicBool::new(false);
```

**Review by bkchr — APPROVED** [review]
Don't we have a test for tracing in wasm?

**Comment by insipx** [comment]
> Don't we have a test for tracing in wasm?

There is this one:
https://github.com/paritytech/substrate/blob/cff25fbc37c9fc564f8816eefdcd8dce15e1606b/client/executor/src/integration_tests/mod.rs#L662

**Comment by bkchr** [comment]
> > Don't we have a test for tracing in wasm?
> 
> There is this one:
> 
> https://github.com/paritytech/substrate/blob/cff25fbc37c9fc564f8816eefdcd8dce15e1606b/client/executor/src/integration_tests/mod.rs#L662
> 
> Guessing it didn't catch this because it only tests one enter/exit span

Can you please adapt it and verify that it failed before and now it works? Ty!

**Comment by insipx** [comment]
here's the result of the changed test:

as const: 



as static:

**Comment by bkchr** [comment]
Nice perfect! Now this error hopefully will not come back!

**Pull request merged**